app/javascript/application.js

```diff
@@ -18,11 +20,14 @@ document.addEventListener('DOMContentLoaded', () => {
       ItemList,
       PurchaseList,
       BaseNotifications,
+      ItemView,
+      InlineSvg,
     },
   });
   app.component('BaseInput', BaseInput);
   app.component('BaseButton', BaseButton);
   app.component('BaseCheckbox', BaseCheckbox);
+  app.component('inline-svg', InlineSvg);
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name "inline-svg" is not PascalCase  vue/component-definition-name-casing

Reply:
Mono 🔪 Esto si lo dejo en PascalCase no me funciona. [En la documentación](https://github.com/shrpne/vue-inline-svg/tree/v3#:~:text=Or%20register%20globally%20in%20the%20Vue%20app) habla de dejarlo así.

Reply:
@vicentematus que raro, [aquí](https://github.com/platanus/mok-services/blob/5135238e8a0481484a7973644a0f45949f8c79a9/app/javascript/application.ts#L82) sí les funciona 🤔. Está con PascalCase [casi todos los proyectos](https://github.com/search?q=org%3Aplatanus%20app.component(%27InlineSvg%27%2C%20InlineSvg)%3B&type=code). Quizás podrías ver cómo llaman al componente en esos proyectos